docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -0,0 +1,520 @@
+---
+title: "Model in image"
+date: 2019-02-23T17:32:31-05:00
+weight: 4
+description: "Sample for supplying a WebLogic Deploy Tool (WDT) model that the operator automatically expands into a full domain home during runtime."
+---
+
+### Contents
+
+  - [Introduction](#introduction)
+  - [References](#references)
+  - [Prerequisites for all domain types](#prerequisites-for-all-domain-types)
+  - [Prerequisites for JRF domains](#prerequisites-for-jrf-domains)
+    - [Set Up and Initialize an RCU Database](#set-up-and-initialize-an-rcu-database)
+    - [Increase introspection job timeout](#increase-introspection-job-timeout)
+    - [Set up RCU model attributes, domain resource attributes, and secrets](#set-up-rcu-model-attributes-domain-resource-attributes-and-secrets)
+    - [Reusing or sharing RCU tables](#reusing-or-sharing-rcu-tables)
+  - [Use the WebLogic Image Tool to create an image](#use-the-weblogic-image-tool-to-create-an-image)
+  - [Create and deploy your Kubernetes resources](#create-and-deploy-your-kubernetes-resources)
+  - [Optionally test the sample application](#optionally-test-the-sample-application)
+  - [Optionally access the WebLogic console](#optionally-access-the-weblogic-console)
+  - [Cleanup](#cleanup)
+
+### Introduction
+
+This sample demonstrates:
+
+  - Using the WebLogic Image Tool to create a Docker image that contains a WebLogic install, a WebLogic Deploy Tool (WDT) install, a Java EE servlet application contained within a WDT archive, and a WebLogic domain that's defined using a WDT model file.
```

Comment:
"WebLogic domain that's defined using a WDT model file"... this is confusing because it specifically doesn't include a WebLogic domain.  Let's say instead, "and the model for a WebLogic domain configuration defined using a WDT model file."

Reply:
will do

Reply:
Done.